include/openthread/srp_server.h

```diff
@@ -308,6 +308,41 @@ otError otSrpServerSetAnycastModeSequenceNumber(otInstance *aInstance, uint8_t a
  */
 void otSrpServerSetEnabled(otInstance *aInstance, bool aEnabled);
```

Comment:
What about updating the documentation to encourage users to use `otSrpServerSetAutoEnableMode` instead?

Reply:
Added it in the new push.